File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        unique = []
        for i in range(len(nums)):
            if nums[i] != val:
                unique.append(nums[i])
        nums[:len(unique)] = unique
        return len(unique)
        

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
#include <iostream>
#include <string>
#include <unordered_map>

class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_set<int> dupli;
        for (int num: nums) {
            if (dupli.count(num)) {
                return true;
            }
            dupli.insert(num);
        }
        return false;
    }
};